client/views/account/accountProfile.html

```diff
@@ -12,7 +12,11 @@
 					<div class="input-line">
 						<label for="username">{{_ "Username"}}</label>
 						<div>
-							<input type="text" name="username" id="username" placeholder="{{username}}" />
+						{{#if allowUsernameChange}}
+						    <input type="text" name="username" id="username" placeholder="{{username}}" />
+						{{else}}
+						    <input type="text" name="username" id="username" placeholder="{{username}}" disabled="disabled" title="{{usernameChangeDisabled}}" />
```

Comment:
Did not know best how to handle the disabling. Here it disables the input field and on mouseover you get a message saying why it is disabled. Might want to remove this which would allow a user to try and change it but they would then get a toastr.error (line 49/50 in `client/views/account/accountProfile.coffee`).